README.md

```diff
@@ -1,34 +1,148 @@
-# Hello World
-Hello World is a project about Introduction to Microverse
-# Built With
-it is built with Html & CSS
-# Key Features
-Html
-css
-# Getting Started
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140" height="auto" />
+  <br/>
+
+  <h3><b>README.md</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+ - [🛠 Built With](#built-with)
+      - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+**[Hello Microverse]** Is a welcome to coding in Microverse
+## 🛠 Built With <a name="built-with"></a>
+    - HTML
+    - CSS
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- #
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>  
+
 To get a local copy up and running, follow these steps.
 
-# Prerequisites
-In order to run this project you need:
-# Setup
-Clone this repository to your desired folder:
-# Install
-Install this project with:
-# Usage
-To run the project, execute the following command:
-# Run tests
-To run tests, run the following command:
-# Deployment
+### Prerequisites
+
+In order to run this project:
+- You need to have a Code Editor Installed
+- Make sure Nodejs and NPM is installed in your system for linters
+
+### Setup
+
+To clone this repository to your desired folder, use your code editor terminal or git bash, then type the command below:
+
+- git clone https://github.com/kiplaacollo/Hello-world.git
+
+
+
+### Install
+
+1. Install WebHint:
+
+npm install --save-dev hint@7.x
+
+2. Install StyleLint
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+
+
+### Usage
+
+To run the project:
+
+- open cloned folder in your code editor
+
+### Run tests
+
+To run tests and check for errors:
+- After installing the required linter,
+
+npx stylelint "**/*.{css,scss}"
+
+- To fix CSS or SCSS linters error:
+
+npx stylelint "**/*.{css,scss}" --fix
+
```

Comment:
- [ ] This section is not necessary for this project. So you will leave the heading as it is but under it, you write `N/A` - Not Applicable.